test.js

```diff
@@ -118,31 +118,31 @@ describe('Concrete Versions', function () {
     it(`compare('1', '1.2.2') throws Error`, function () {
       assume(function () {
         conver.compare('1', '1.2.2');
-      }).throws('Inputs contains illegal semver');
+      }).throws('contain illegal semver');
```

Comment:
Any reason to not put the full error message here like you did in the others?

Reply:
Was failing with an error I could not diagnose:

```
  1) Concrete Versions
       compare
         compare('1', '1.2.2') throws Error:
     Unknown assertation failure occured, assumed `'Inputs { 1, 1.2.2 } contain illegal semver'` to include 'Inputs { 1, 1.2.2 } contain illegal semver'
  /Users/crobbins/Git/indexzero/conver/test.js
                    v
      120.         conver.compare('1', '1.2.2');
      121.       }).throws('Inputs { 1, 1.2.2 } contain illegal semver');
      122.     });
                    ^
          at Assert.throws (node_modules/assume/index.js:706:49)
          at Context.<anonymous> (test.js:121:10)
```

The strings are equal which is the part I don't get.